README.md

```diff
@@ -1,2 +1,175 @@
-# ApplitoolsIntegration
-Utility to easily integrate Applitools VisualAI with your automation framework
+# Introduction
+
+1. Utility to easily integrate Applitools VisualAI with your JAVA-TestNG automation framework
+2. This is a library which implements basic utilities
+   for [Applitools](https://applitools.com/docs/topics/overview.html) `(com.applitools)`
+3. Dashboard link: https://eyes.applitools.com/
+
+# Build
+
+`mvn clean install -DskipTests`
+> If facing issues with dependencies not being resolved from https://jitpack.io, then check the `settings.xml` file
+> you're using for building your maven projects. If you've proxies configured in the same, then make sure `jitpack.io`
+> is part of `nonProxyHosts` configuration. For instance
+
+```xml
+
+<proxy>
+    <id>httpmyproxy</id>
+    <active>true</active>
+    <protocol>http</protocol>
+    <host>someHost</host>
+    <port>8080</port>
+    <username>UserName</username>
+    <password>Password</password>
+    <nonProxyHosts>*.google.com|*jitpack.io</nonProxyHosts>
+</proxy>
+```
+
+# How to consume the dependency
+
+   ```
+   <dependency>
+      <groupId>com.github.znsio</groupId>
+      <artifactId>ApplitoolsIntegration</artifactId>
+      <version>x.x.x</version>
+   </dependency>
+   ```
+
+# How to configure Applitools (Visual Testing) in your automation framework
+
+1. To disable visual validation for the tests, set `IS_VISUAL` to `false` in config properties of your automation
+   framework repo
+2. For running visual validations:
+    1. Set `IS_VISUAL` to `true` in config properties of your automation framework repo
+    2. Set the environment variable for `APPLITOOLS_API_KEY` like this
+       ```
+       export APPLITOOLS_API_KEY=<Your API KEY>
+       ```
+       Applitools API Key can be picked from user Dashboard `https://eyes.applitools.com/`
+    3. **Properties file:** Create a `applitools.properties` file either in `src/test/resources` directory in your
+       automation framework. Specify the path of the file in you `config.properties` of your automation framework like
+       this:
+       ```
+       APPLITOOLS_CONFIGURATION_FILE=./src/test/resources/applitools.properties
+       ```
+       The `applitools.properties` file should have the following mandatory attributes:
+       ```
+       SERVER_URL=https://eyes.applitools.com/
+       CONCURRENCY=<Integer Value>
+       MATCH_LEVEL=strict
+       SEND_DOM=<true or false>
+       STITCH_MODE=css
+       TAKE_FULL_PAGE_SCREENSHOT=<true or false>
+       VIEWPORT_SIZE=1200x700
+       USE_UFG=<true or false>
+       SHOW_LOGS=<true or false>
+       SAVE_BASELINE_FOR_NEW_TESTS=<true or false>
+       ```
+    4. Annotate the test with the `visual` TestNG group annotation which you want to run as part of visual validation.
```

Comment:
add an example here - maybe a screenshot of a test with this `visual` annotation

Reply:
Added